common/testData/localeIdentifiers/likelySubtags.txt

```diff
@@ -0,0 +1,1576 @@
+sr-RS ;	sr-Cyrl-RS ;	sr ;	
+sr-XK ;	sr-Cyrl-XK ;	sr-XK ;	
+su ;	su-Latn-ID ;	su ;	
+su-AQ ;	su-Latn-AQ ;	su-AQ ;	
+su-Egyp ;	su-Egyp-ID ;	su-Egyp ;	
+su-ID ;	su-Latn-ID ;	su ;	
+su-Latn ;	su-Latn-ID ;	su ;	
+sv ;	sv-Latn-SE ;	sv ;	
+sv-AQ ;	sv-Latn-AQ ;	sv-AQ ;	
+sv-AX ;	sv-Latn-AX ;	sv-AX ;	
+sv-Egyp ;	sv-Egyp-SE ;	sv-Egyp ;	
+sv-FI ;	sv-Latn-FI ;	sv-FI ;	
+sv-Latn ;	sv-Latn-SE ;	sv ;	
+sv-SE ;	sv-Latn-SE ;	sv ;	
+sw ;	sw-Latn-TZ ;	sw ;	
+sw-AQ ;	sw-Latn-AQ ;	sw-AQ ;	
+sw-CD ;	sw-Latn-CD ;	sw-CD ;	
+sw-Egyp ;	sw-Egyp-TZ ;	sw-Egyp ;	
+sw-KE ;	sw-Latn-KE ;	sw-KE ;	
+sw-Latn ;	sw-Latn-TZ ;	sw ;	
+sw-TZ ;	sw-Latn-TZ ;	sw ;	
+sw-UG ;	sw-Latn-UG ;	sw-UG ;	
+ta ;	ta-Taml-IN ;	ta ;	
+ta-AQ ;	ta-Taml-AQ ;	ta-AQ ;	
+ta-Egyp ;	ta-Egyp-IN ;	ta-Egyp ;	
+ta-IN ;	ta-Taml-IN ;	ta ;	
+ta-LK ;	ta-Taml-LK ;	ta-LK ;	
+ta-MY ;	ta-Taml-MY ;	ta-MY ;	
+ta-SG ;	ta-Taml-SG ;	ta-SG ;	
+ta-Taml ;	ta-Taml-IN ;	ta ;	
+te ;	te-Telu-IN ;	te ;	
+te-AQ ;	te-Telu-AQ ;	te-AQ ;	
+te-Egyp ;	te-Egyp-IN ;	te-Egyp ;	
+te-IN ;	te-Telu-IN ;	te ;	
+te-Telu ;	te-Telu-IN ;	te ;	
+tg ;	tg-Cyrl-TJ ;	tg ;	
+tg-AQ ;	tg-Cyrl-AQ ;	tg-AQ ;	
+tg-Cyrl ;	tg-Cyrl-TJ ;	tg ;	
+tg-Egyp ;	tg-Egyp-TJ ;	tg-Egyp ;	
+tg-TJ ;	tg-Cyrl-TJ ;	tg ;	
+th ;	th-Thai-TH ;	th ;	
+th-AQ ;	th-Thai-AQ ;	th-AQ ;	
+th-Egyp ;	th-Egyp-TH ;	th-Egyp ;	
+th-TH ;	th-Thai-TH ;	th ;	
+th-Thai ;	th-Thai-TH ;	th ;	
+ti ;	ti-Ethi-ET ;	ti ;	
+ti-AQ ;	ti-Ethi-AQ ;	ti-AQ ;	
+ti-ER ;	ti-Ethi-ER ;	ti-ER ;	
+ti-ET ;	ti-Ethi-ET ;	ti ;	
+ti-Egyp ;	ti-Egyp-ET ;	ti-Egyp ;	
+ti-Ethi ;	ti-Ethi-ET ;	ti ;	
+tk ;	tk-Latn-TM ;	tk ;	
+tk-AQ ;	tk-Latn-AQ ;	tk-AQ ;	
+tk-Egyp ;	tk-Egyp-TM ;	tk-Egyp ;	
+tk-Latn ;	tk-Latn-TM ;	tk ;	
+tk-TM ;	tk-Latn-TM ;	tk ;	
+to ;	to-Latn-TO ;	to ;	
+to-AQ ;	to-Latn-AQ ;	to-AQ ;	
+to-Egyp ;	to-Egyp-TO ;	to-Egyp ;	
+to-Latn ;	to-Latn-TO ;	to ;	
+to-TO ;	to-Latn-TO ;	to ;	
+tr ;	tr-Latn-TR ;	tr ;	
+tr-AQ ;	tr-Latn-AQ ;	tr-AQ ;	
+tr-CY ;	tr-Latn-CY ;	tr-CY ;	
+tr-Egyp ;	tr-Egyp-TR ;	tr-Egyp ;	
+tr-Latn ;	tr-Latn-TR ;	tr ;	
+tr-TR ;	tr-Latn-TR ;	tr ;	
+tt ;	tt-Cyrl-RU ;	tt ;	
+tt-AQ ;	tt-Cyrl-AQ ;	tt-AQ ;	
+tt-Cyrl ;	tt-Cyrl-RU ;	tt ;	
+tt-Egyp ;	tt-Egyp-RU ;	tt-Egyp ;	
+tt-RU ;	tt-Cyrl-RU ;	tt ;	
+uk ;	uk-Cyrl-UA ;	uk ;	
+uk-AQ ;	uk-Cyrl-AQ ;	uk-AQ ;	
+uk-Cyrl ;	uk-Cyrl-UA ;	uk ;	
+uk-Egyp ;	uk-Egyp-UA ;	uk-Egyp ;	
+uk-UA ;	uk-Cyrl-UA ;	uk ;	
+und-001 ;	en-Latn-001 ;	en-001 ;	
+und-150 ;	ru-Cyrl-RU ;	ru ;	
+und-419 ;	es-Latn-419 ;	es-419 ;	
+und-AD ;	ca-Latn-AD ;	ca-AD ;	
+und-AE ;	ar-Arab-AE ;	ar-AE ;	
+und-AF ;	fa-Arab-AF ;	fa-AF ;	
+und-AG ;	en-Latn-AG ;	en-AG ;	
+und-AI ;	en-Latn-AI ;	en-AI ;	
+und-AL ;	sq-Latn-AL ;	sq ;	
+und-AM ;	hy-Armn-AM ;	hy ;	
+und-AO ;	pt-Latn-AO ;	pt-AO ;	
+und-AQ ;	und-Latn-AQ ;	und-AQ ;	
+und-AR ;	es-Latn-AR ;	es-AR ;	
+und-AS ;	sm-Latn-AS ;	sm-AS ;	
+und-AT ;	de-Latn-AT ;	de-AT ;	
+und-AU ;	en-Latn-AU ;	en-AU ;	
+und-AW ;	nl-Latn-AW ;	nl-AW ;	
+und-AX ;	sv-Latn-AX ;	sv-AX ;	
+und-AZ ;	az-Latn-AZ ;	az ;	
+und-Adlm ;	ff-Adlm-GN ;	ff-Adlm ;	
+und-Adlm-BF ;	fr-Adlm-BF ;	 ;	
```

Comment:
Could you please explain what does  the RemoveFavorRegion = "" mean?

Reply:
It's in the header. In other words, if a value is missing, it means the same as the previous field in the row (recursively).

```
#   RemoveFavorScript: Remove Likely Subtags, when the script is favored.
#                      Only included when different than AddLikely.
#   RemoveFavorRegion: Remove Likely Subtags, when the region is favored.
#                      Only included when different than RemoveFavorScript.
```
If that isn't clear we can improve the header.